### DISCUSSION TITLE
My nextJs app generates SSR pages but when viewing page src it's only showing the main app div with other scripts, like it's doing CSR!

### DISCUSSION THREAD

**Discussion by rem0nfawzi** [vercel/next.js#16349] [Help]
My nextJs app generates SSR pages but when viewing page src it's only showing the main app div with other scripts, like it's doing CSR! and this happens with all pages.

I'm using getInitialProps in my _app.js file.

My nextJS version is 9.4.4

**Comment by IRediTOTO** [comment]
I think i got this error like you before, but it from my code. Let's check app.js. Some wrapper component make them not render on SS.

**Comment by Nora-group** [comment]
Okay ,So I got your Problem as you might used  getServerSideProps() instead of getStaticProps()  do replace them
